pkgs/servers/web-apps/snipe-it/default.nix

```diff
@@ -18,13 +18,13 @@ let
 
 in package.override rec {
   pname = "snipe-it";
-  version = "6.0.5";
+  version = "6.0.6";
```

Comment:
```suggestion
  version = "6.0.7";
```

Reply:
Ah, thanks for the hint! Updating the PR...